cranelift-codegen/shared/Cargo.toml

```diff
@@ -0,0 +1,11 @@
+[package]
+authors = ["The Cranelift Project Developers"]
+name = "cranelift-codegen-shared"
```

Comment:
It's a thing \o/

We might need to update the top-level `./publish-all.sh` script around lines 18 and 41.